taskcluster/mozillavpn_taskgraph/target_tasks.py

```diff
@@ -5,13 +5,69 @@
 from taskgraph.target_tasks import _target_task
 
 
-@_target_task("nightly")
-def target_tasks_nightly(full_task_graph, parameters, graph_config):
-    """Select the set of tasks required for a nightly build."""
+@_target_task("promote-addons")
+def target_tasks_promote_addons(full_task_graph, parameters, graph_config):
+    def filter(task):
+        if (
+            task.attributes.get("shipping-phase") == "promote-addons"
+            and task.attributes.get("build-type") == "addons/opt"
```

Comment:
Doesn't hurt to have this `build-type` check, though it seems like the first check should be enough.